src/ensembl/src/content/app/entity-viewer/gene-view/components/protein-domain-image/ProteinDomainImage.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { createTranscript } from 'tests/fixtures/entity-viewer/transcript';
+
+import ProteinDomainImage, {
+  getDomainsByResourceGroups
+} from './ProteinDomainImage';
+
+const minimalProps = {
+  transcript: createTranscript(),
+  width: 600
+};
+
+const domainsByResourceGroups = getDomainsByResourceGroups(
+  minimalProps.transcript.product.protein_domains_resources
+);
+
+describe('<ProteinDomainImage />', () => {
+  let wrapper: any;
+
+  beforeEach(() => {
+    wrapper = mount(<ProteinDomainImage {...minimalProps} />);
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it(' renders N SVGs based on the number of protein domains', () => {
```

Comment:
`it('renders the correct number of tracks')` (an `svg` tag in your code represents a track with domains in it)

Reply:
That's done